portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,37 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.CommentData;
```

Comment:
Prefer imports in alphabetical order

Reply:
Fixed in commit "Put imports in alphabetical order" !

Reply:
I have fixed this. Thanks!